src/core/toxcall.cpp

```diff
@@ -187,8 +193,14 @@ void ToxFriendCall::setState(const TOXAV_FRIEND_CALL_STATE& value)
     state = value;
 }
 
+const std::unique_ptr<IAudioSink> &ToxFriendCall::getAudioSink() const
```

Comment:
I'm not sure, that it's good idea to get `unique_ptr` reference. Why not just reference on `IAudioSink`?

Also probably `&` must be closer to type

Reply:
Also. this violates the "Law of Demeter". Also, interesting article about "Tell, Don't Ask" principle: https://pragprog.com/articles/tell-dont-ask